src/components/CryptoCards/types.ts

```diff
@@ -0,0 +1,43 @@
+import { iconTypes } from '../Icon/collection';
+
+export interface CryptoCardProps {
+    /**
+     * A function that will be called if the button is clicked
+     */
+    buttonClickEvent?: () => void;
+
+    /**
+     * The name of the blockchain
+     */
+    chain: string;
+
+    /**
+     * The type of the chain / a subtitle below the chain name
+     */
+    chainType: string;
+
+    /**
+     * The chain logo
+     */
+    chainLogo: string;
```

Comment:
I'm not sure if we should just stick to the chains we have. @BillyG83 what do you think about :

`chain: "eth" | "bsc" | "fantom" | "polygon" | "arbitrum"` etc etc.

Reply:
done